test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -34,7 +34,8 @@ namespace dev { namespace test {
 bool createRandomTest()
 {
 	StateTestSuite suite;
-	dev::test::Options& options = const_cast<dev::test::Options&>(dev::test::Options::get());
+	dev::test::Options& optionsConst = const_cast<dev::test::Options&>(dev::test::Options::get());
```

Comment:
no need for cast here at all. please use dev::test::Options const&   here

Reply:
Sorry, you'll have to ELI5 (and don't know c++). The `const_cast` was already there https://github.com/ethereum/cpp-ethereum/blob/develop/test/tools/fuzzTesting/createRandomTest.cpp#L37